src/Admin.php

```diff
@@ -25,4 +25,89 @@ public static function admin_notices() {
     <?php }
   }
 
+  /**
+   * Returns products backlinking to a given accessory product.
+   */
+  public static function wp_ajax_related_accessories_backlinks() {
+    global $wpdb;
+    $postId = intval($_POST['post']);
+    $html = '<ol>';
+    if($postId)
+    {
+      $params = [
+        'post_type' => ['product', 'product_variant'],
+        'post_status' => 'any',
+        'meta_query' => [
+          ['key' => 'field_group_related_accessories_care_products',
+            'value' => '"' . $postId . '"',
+            'compare' => 'LIKE',
+          ],
+        ],
+        'posts_per_page' => -1,
+      ];
+      $wc_query = new \WP_Query($params);
+      global $post, $product;
+      if ($wc_query->have_posts()) {
+        while ($wc_query->have_posts()) {
+          $wc_query->the_post();
+          $html .= '<li><a href="'.get_edit_post_link($wc_query->post).'">';
+          $html .= esc_html($wc_query->post->post_title);
+          $html .= '</a></li>';
```

Comment:
1. Did you check how the ACF references widget renders these entries (including a nice thumbnail)? Maybe we can reuse the function of ACF, as this plugin depends on it anyway?

2. Coding style violation: There should be a space before and after the string concatenation operator (in all languages, not only PHP).

Reply:
I carefully considered the ACF references widget:
- sometimes a list of more than 100 products is populated, adding the thumbnail handlers might have a performance tax
- the simple ordered product link list was already validated by the PM as suficcient on the stagaing, and this thumbnail feature was not described in the ticket nor estimated. 
- Further more it will introduce more complication to the code than necessary for the functionality requested.